docs/src/main/asciidoc/_migration-guide-for-6.0.adoc

```diff
@@ -0,0 +1,145 @@
+
+.Deleted artifact Id
+[cols="<,<,<", options="header"]
+|===
+|Legacy Artifact ID |Modern Artifact ID |Description
+|spring-cloud-azure-starter-data-cosmos |N/A |Delete this artifact because the Spring Data 3 adaptation has not been completed.
+|===
+
+=== Dependency Changes
+
+The modern Spring Cloud Azure 6.0 libraries are based on Jakarta EE 9, which uses a new top-level jakarta package, it's required to replace EE 8’s javax top-level package.
+Please make sure to correct the `javax` package to `jakarta` package in your application.
+
+Libraries that have dependency changes include:
+
+- <<dependency-spring-cloud-azure-starter-active-directory, spring-cloud-azure-starter-active-directory>>
+- <<dependency-spring-cloud-azure-starter-active-directory-b2c, spring-cloud-azure-starter-active-directory-b2c>>
+- <<dependency-spring-cloud-azure-starter-servicebus-jms, spring-cloud-azure-starter-servicebus-jms>>
+
+=== API breaking changes
+
+For API breaking changes in each libraries, please refer to below links for details.
+
+- <<api-spring-cloud-azure-starter-data-cosmos, spring-cloud-azure-starter-data-cosmos>>
+
+=== Library Changes
+
+Breaking changes in each library are introduced as follows.
+
+==== spring-cloud-azure-starter-active-directory
+
+* Please refer to <<migration-guide-introduction, introduction>> and <<migration-guide-benefits, migration benefits>> to get a whole picture of the changes in 6.0.
+
+[#dependency-spring-cloud-azure-starter-active-directory]
+===== Dependency Changes
+
+Some dependencies in the legacy artifact have been upgraded since the modern Spring Cloud Azure 6.0 library. Please update these changes to your project to fix compilation errors.
+
+.Upgrade Dependencies
+[cols="<,<,<", options="header"]
+|===
+|Legacy dependency |Modern dependency |Description
+|javax.servlet:javax.servlet-api
+|jakarta.servlet:jakarta.servlet-api
+|Required.
+|===
+
+==== spring-cloud-azure-starter-active-directory-b2c
+
+* Please refer to <<migration-guide-introduction, introduction>> and <<migration-guide-benefits, migration benefits>> to get a whole picture of the changes in 6.0.
+
+[#dependency-spring-cloud-azure-starter-active-directory-b2c]
+===== Dependency Changes
+
+Some dependencies in the legacy artifact have been upgraded since the modern Spring Cloud Azure 6.0 library.
+Please update these changes to your project to fix compilation errors.
+
+.Upgrade Dependencies
+[cols="<,<,<", options="header"]
+|===
+|Legacy dependency |Modern dependency |Description
+|javax.servlet:javax.servlet-api
+|jakarta.servlet:jakarta.servlet-api
+|Required.
+|===
+
+==== spring-cloud-azure-starter-data-cosmos
+
+The adaptation of Spring Data 3 has not been completed, Spring Cloud Azure 6.0 will not include the Spring Data Cosmos part,
+and the related API has been deleted.
+
+[#api-spring-cloud-azure-starter-data-cosmos]
+===== API Changes
```

Comment:
It is not necessary if artifact (`spring-cloud-azure-starter-data-cosmos`) not exists in version 6.0.

Reply:
OK